ShellPkg/Library/UefiShellAcpiViewCommandLib/Parsers/Fpdt/FpdtParser.c

```diff
@@ -0,0 +1,263 @@
+/** @file
+  Firmware Performance Data Table (FPDT) table parser
+
+  Copyright (C) 2025, Advanced Micro Devices, Inc. All rights reserved.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+  @par Reference(s):
+    - ACPI 6.5 Specification - August 2022
+**/
+
+#include <IndustryStandard/Acpi.h>
+#include <Library/UefiLib.h>
+#include <Library/DebugLib.h>
+#include "AcpiParser.h"
+#include "AcpiTableParser.h"
+#include "AcpiViewConfig.h"
+
+// Local Variables
+STATIC ACPI_DESCRIPTION_HEADER_INFO  AcpiHdrInfo;

```

Comment:
Should be `mAcpiHdrInfo` for static structures

Reply:
It has not been strictly applied to other acpi parsers, but normally this should apply to all static structures.
It seems there are other static structures that remain without the `m` prefix.